WARNING | Неверный тип данных для объекта Runner
Traceback (most recent call last):
  File "C:\programming\PythonProjectsForUnivesity\lesson0\lesson12\tests_12_4.py", line 26, in test_run
    leo = rnr.Runner(23, 5)
          ^^^^^^^^^^^^^^^^^
  File "C:\programming\PythonProjectsForUnivesity\lesson0\lesson12\rt_with_exceptions.py", line 6, in __init__
    raise TypeError(f'Имя может быть только строкой, передано {type(name).__name__}')
TypeError: Имя может быть только строкой, передано int
WARNING | Неверная скорость для Runner
Traceback (most recent call last):
  File "C:\programming\PythonProjectsForUnivesity\lesson0\lesson12\tests_12_4.py", line 12, in test_walk
    ole = rnr.Runner('Ole', -10)
          ^^^^^^^^^^^^^^^^^^^^^^
  File "C:\programming\PythonProjectsForUnivesity\lesson0\lesson12\rt_with_exceptions.py", line 11, in __init__
    raise ValueError(f'Скорость не может быть отрицательной, сейчас {speed}')
ValueError: Скорость не может быть отрицательной, сейчас -10
